description.txt

```diff
@@ -1,7 +1,7 @@
 No more wrestling with Java plug-in errors, computer incompatibilities, or user flows that leave you confused. And no more proprietary data silos. The Tidepool Uploader works with Mac and PC.
 
 ■ Supported devices
-Dexcom G4 Platinum, Insulet OmniPod, Medtronic Minimed (523 and up), Abbott Precision Xtra.
+Dexcom (G4 Platinum and G5), Insulet OmniPod, Medtronic Minimed (523 and up), Abbott Precision Xtra.
```

Comment:
Want to touch base w/product about whether we want to update this further to include the other currently supported BG meters?